contracts/token/ERC721/ERC721.sol

```diff
@@ -209,14 +209,24 @@ contract ERC721 is ERC165, IERC721 {
     /**
      * @dev Internal function to burn a specific token
      * Reverts if the token does not exist
-     * @param tokenId uint256 ID of the token being burned by the msg.sender
+     * Deprecated, use _burn(uint256) instead.
+     * @param tokenId uint256 ID of the token being burned
```

Comment:
Can you add the missing `@param owner` here? :grimacing:

Reply:
:grimacing: